lib/kitchen/directions/bake_references/v1.rb

```diff
@@ -28,12 +28,14 @@ def bake(book:, metadata_source:)
         end
 
         chapter_references = chapter.pages.references.cut
-        chapter_title_no_num = chapter.title.search('.os-text')
+        chapter_title = chapter.title.search('.os-text')
+
+        chapter_title = chapter.title.search('.os-number, .os-divider, .os-text') if numbered_title
```

Comment:
I'm wondering if it's ok to have two definitions :thinking: What do you think to use `if else`?

```
chapter_title = if numbered_title do
                          chapter.title.search('.os-number, .os-divider, .os-text')
                       else
                          chapter.title.search('.os-text')
                       end
```